metricbeat/metricbeat.yml

```diff
@@ -6,37 +6,17 @@
 #
 # You can find the full configuration reference here:
 # https://www.elastic.co/guide/en/beats/metricbeat/index.html
-
 #==========================  Modules configuration ============================
-metricbeat.modules:
-
-#------------------------------- System Module -------------------------------
-- module: system
-  enabled: true
-  period: 10s
-  metricsets:
-    - cpu
-    - load
-    - memory
-    #- core
-    #- diskio
-    - network
-    - process_summary
-    - process
-    #- socket
-  processes: ['.*']
-  process.include_top_n:
-    by_cpu: 5      # include top 5 processes by CPU
-    by_memory: 5   # include top 5 processes by memory
-- module: system
-  enabled: true
-  period: 1m
-  metricsets:
-    - filesystem
-    - fsstat
-  filters:
-    - drop_event.when.regexp.mount_point: '^/(sys|cgroup|proc|dev|etc|host|lib)($|/)'
 
+metricbeat.config.modules:
+  # Glob pattern for configuration loading
+  path: ${path.config}/modules.d/*.yml
```

Comment:
I guess most people use downloaded packages or tar.gz, everything will be on those.

For the scripted case, old config formats should still work. I don't see this becoming an issue but you never know, it's a good point